diagnostic.md

```diff
@@ -36,13 +40,17 @@ Now, we want to get a single donut from the server. Write the `curl` request
 you'd use to retrieve a single donut, using whatever ID you'd like.
 
 ```sh
-# your answer here
+curl --request GET http://www.example.com/donuts/3
 ```
 
 Write an AJAX request to retrieve a single donut from the server.
 
 ```js
-let getDonut = /* your answer here */;
+let getDonut = function() {
+  $.ajax({
+    url: 'http://www.example.com/donuts/3',
+    type: 'GET'
+  });
```

Comment:
As above, you didn't close the function, and you haven't set the `.done` and `.fail` handlers.